dash/components/src/Chrome/SidebarNav.tsx

```diff
@@ -79,6 +80,15 @@ const SidebarNav: React.FC<Props> = ({
         >
           Keychains
         </SidebarOption>
+        <SidebarOption
+          Icon={ShieldCheckIcon}
+          collapsed={collapsed}
+          to="/security-events"
+          onClick={onInternalLinkClick}
+          isSelected={urlPath.startsWith(`/security-events`)}
+        >
+          Security
+        </SidebarOption>
```

Comment:
this looks nice, thanks. however, can you comment it out for now? i'd like to use the feature myself for a week or two before exposing the link. and i'm not 100% sure it should be a top-level sidebar link yet either. but if it does become one, this looks great, so just comment it out for now, i think. 👍